### ISSUE TITLE
Screenshot is "half width" regardless of viewport

### ISSUE THREAD

**Issue by khrome83** [cypress-io/cypress#2018]
### Current behavior:
When taking a screenshot, the width is half width. I have not modified any default except to add a onAfterScreenshot callback.

```
  cy.viewport('iphone-6+');
  cy.screenshot(name, {
    onAfterScreenshot: ($el, props) => {
      screenshots.push(props.path);
    }
  })
```
See example - 

![homepage](https://user-images.githubusercontent.com/1844739/41789159-f1222b02-7613-11e8-8bfa-c4851a120cf5.png)

### Desired behavior:

Screenshot shows the full width of the viewport.

### Steps to reproduce:

Code snippet above.

### Versions

Cypress - 3.0.1
Chrome - 67
OSX - 10.13.5

**Comment by chrisbreiding** [comment]
This is likely the same issue as #1857. Screenshots taken on a high DPI screen are not being handled correctly. This is already fixed and will be released in [3.0.2](https://github.com/cypress-io/cypress/milestone/77), which should be out in the near future.

**Comment by brian-mann** [comment]
Released in [`3.0.2`](https://docs.cypress.io/guides/references/changelog.html#3-0-2).
